lib/common/utils.ts

```diff
@@ -11,50 +11,51 @@
  * @suppress {undefinedVars,globalThis,missingRequire}
  */
 
+// issue #989, to reduce bundle size, use short name
+export const a = Object.getOwnPropertyDescriptor;
```

Comment:
there is no need to have a single char letters here. You can have long names, they will get renamed using uglify. Single letters are hard on readability.

Reply:
Thanks for pointing out this one, I have misunderstanding about uglfiyjs, I have changed the variable name to long names.